blog/_posts/Hola, Amigos.md

```diff
@@ -0,0 +1,118 @@
+
+### Rupal
+"I am a junior majoring in *Computer Science* and minoring in *Computational Biology*. This is what my profile speaks, how I want to describe myself as is: A curious maniac, who has (mysteriously) learned the art of keeping herself stimulated and keeping up with whatever I do. A programmer, a reader, an extraordinarily passionate bathroom dancer, and a diehard *Brooklyn Nine Nine* fan, I am a keen knowledge seeking individual whose patience and grit accelerates her ability to articulate difficult concepts and texts smoothly. I’d describe myself as a strong woman who has the courage to voice her opinions, someone who is never hesitant to take actions. I’m one of those people who are if in a conference room and the projector stops working, I’m not the sort to simply call IT and wait. I’ll just (gracefully) crawl under the table and check that everything is properly plugged in.
+
+I am also a sports enthusiast, with basketball being one of my favourite sports. A book in hand, headphones and a decent internet connection is all I need when in melancholy. Quora comes to the rescue when the critic in me tends to rise.
+
+Elevating Medical Sciences using computers and technology is my passion, and it will be my future. I want to become a Computer Scientist and be known for my talent, and not because I’m a *woman*. I want people to understand that *I am more than what they can see*."
+
+I have had this habit of imaginng things in my head, things that I aspire to do and achieve, but the results are not always the way I expected them to be. But then, I take a deep breath and reflect back, and am reminded of the famous quote from the *Harry Potter series*
+
+> *“Of course it is happening inside your head, Harry, but why on earth should that mean that it is not real?” ― Albus Dumbledore, Harry Potter and the Deathly Hallows*
+
+But this is just a way to pacify myself and keep dreaming with a head held high. :D
+
+## Our Team
+
+### Our mentor:
+[**Michael Ferguson**](https://github.com/mppf) and [**Ben Albrecht**](https://github.com/ben-albrecht):
+
+We are delighted to have them as our mentors for the project, and hope that under their able guidance, we can have a steep learning curve. Also, we are grateful to them for helping us during the application process.
+
+### Our Official coaches:
+We are grateful to them for agreeing to coach us, guide us, and help us grow and learn, as we embark upon this journey. Again, we are extremely thankful to him for all the mentoring at the time of applications.
+
+ - [**Louis Jenkins**](https://louisjenkinscs.github.io/): He is a  recent Computer Science graduate from Bloomsburg interests and experience in the low-level areas such as compilers, operating systems, and the design and implementation of high-performance data structures. [Source](https://louisjenkinscs.github.io/)
```

Comment:
Not sure why you have added the same website/linkedin links twice. Please remove the links with 'Source' texts.